janus/entrypoint.sh

```diff
@@ -11,5 +11,6 @@ if [[ -z ${LOCAL_IP} ]]; then
 	sed -i "s/\t#local_media_ip = .*/\tlocal_media_ip = \"${LOCAL_IP}\"/" /usr/local/etc/janus/janus.plugin.sip.jcfg
 fi
 sed -i "s/\tport = .*/\tport = \"${JANUS_TRANSPORT_PORT:=8089}\"/" /usr/local/etc/janus/janus.transport.http.jcfg
+sed -i "s/.*rtp_port_range = .*/\trtp_port_range = \"${ASTERISK_RTPSTART}-${ASTERISK_RTPEND}\"/" janus/usr/local/etc/janus/janus.plugin.sip.jcfg
```

Comment:
The range is already set here: https://github.com/nethesis/ns8-nethvoice/commit/5e3b18d72c15cc9e09185eba4100cd58bc6ff3d1#diff-270d9679bebe4a422bb5d539fe8cea6858b7a93122169b34344e770bb17dee05R32

Reply:
are you sure? reading the doc, it seems the first one is used also for other rtp ranges, if not overridden
but I may be wrong, the doc is not really clear

Reply:
I can't understand it by looking at the code, probably you're right. I'll remove it from the pr and test without it. We can add it later if it was needed